app/assets/javascripts/quotingPanel.js

```diff
@@ -56,22 +69,38 @@ class QuotingPanel {
     return totalAreaInMts * constants.SQ_FT_CONVERT;
   }
 
-  calculateTotalDue() {
+  updateAreaNode() {
+    const areaNode = document.getElementById("calculatedArea");
+    areaNode.innerText = `${this.totalAreaInSqFt.toLocaleString(undefined, {maximumFractionDigits: 0})}`;
+  }
+
+  calculateSubTotal() {
     return this.totalAreaInSqFt * constants.PRICE_PER_SQ_FT;
   }
 
+  updateSubTotalNode() {
+    const subTotalNode = document.getElementById("subTotalDue");
+    subTotalNode.innerText = `${(this.subTotal).toLocaleString(undefined, {maximumFractionDigits: 2})}`;
+  }
+
+  updateTotalSaltBagsNode() {
+    const numberOfBagsNode = document.getElementById("numberOfBags");
+    numberOfBagsNode.innerText = `${this.saltBagsQuantity}`;
+    const bagsDueNode = document.getElementById("saltBagsDue");
+    bagsDueNode.innerText = `${this.saltBagsDue}`;
+    this.updateTotalDueNode();
+  }
+
   showTotalsNode() {
     document.getElementById("displayTotals").classList.remove("hidden");
   }
   
-  updateAreaNode() {
-    const areaNode = document.getElementById("calculatedArea");
-    areaNode.innerText = `${this.totalAreaInSqFt.toLocaleString(undefined, {maximumFractionDigits: 0})}`;
-  }
-
-  updateTotalDueNode() {
-    const areaNodeInModal = document.getElementById("areaModal");
-    areaNodeInModal.innerText = `${this.totalAreaInSqFt.toLocaleString(undefined, {maximumFractionDigits: 0})}`;
+  calculateTotalDue() {
+    let totalDue = this.subTotal + this.serviceExpeditionCost + this.saltBagsDue;
```

Comment:
Confirm that `saltBagsDue` is not _on top_ of the minimum charge

Reply:
will check with Snow Queen